src/resolvers/resolver.cr

```diff
@@ -44,7 +44,7 @@ module Shards
       end
     end
 
-    protected def install_path
+    def install_path
```

Comment:
This is an application, not a library. You may use it as a library, but at your own risk: I don't guarantee the internal API to be stable.